runners/core-java/src/main/java/org/apache/beam/runners/core/metrics/MetricsContainerImpl.java

```diff
@@ -357,6 +367,14 @@ public Map<String, ByteString> getMonitoringData(ShortIdMap shortIds) {
         builder.put(shortId, encodeInt64Distribution(metricUpdate.getUpdate()));
       }
     }
+    for (MetricUpdate<org.apache.beam.runners.core.metrics.GaugeData> metricUpdate :
```

Comment:
nit: s/org.apache.beam.runners.core.metrics.GaugeData/Guage

Reply:
Good catch. Somehow it got autoformatted..

Reply:
Wait... I don't understand this comment.

Reply:
oh sed comment.

Reply:
fixed